### ISSUE TITLE
[REQ] Add support for extracting Debian packages (.deb)

### ISSUE THREAD

**Issue by Bendodroid** [aonez/Keka#1498]
### Is your feature request related to a problem? Please describe
I often work with .deb files and they just appear as the white-page-unkown-file-type-placeholder icon in the Finder.

### Describe the solution you'd like
.deb files are just compressed archives with a well-defined layout. I would like to have a nice file icon when Keka is registered as the default uncompressor and for Keka to completely decompress .deb files so I can inspect if the file layout/contents are correct.

### Describe alternatives you've considered
I use tools on the command line to achieve the desired effect, but double-clicking the file in the Finder and Keka decompressing alle the nested archives in one go would save time.

**Comment by gingerbeardman** [comment]
What command line tools can work with them on macOS?

**Comment by Bendodroid** [comment]
> What command line tools can work with them on macOS?

Yes, I use `ar` for the first layer, then there is the second layer of regular `tar` stuff for which I can already use Keka.
